src/main/java/seedu/address/logic/commands/FindClassCodeCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.ClassCodeContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose class code matches the given class code.
+ * Keyword matching is case insensitive.
+ */
+public class FindClassCodeCommand extends Command {
+
+    public static final String COMMAND_WORD = "findclasscode";
```

Comment:
Should the COMMAND_WORD be find_by_class to follow our user guide?
```suggestion
    public static final String COMMAND_WORD = "find_by_class";
```

Reply:
I would recommend sticking with the previous convention as the user guide is subjected to change